polaris-react/src/components/DescriptionList/DescriptionList.scss

```
@@ -1,13 +1,13 @@
 @import '../../styles/common';
 
-$breakpoint: 550px;
+$magic-breakpoints-up: '(min-width: 36.875em)'; // 590px
```

```suggestion
$legacy-breakpoints-up: '(min-width: 36.875em)'; // 590px
```

What do you think about legacy?

---

Works for me. Was following the existing trend in [_layout.scss](https://github.com/Shopify/polaris/blob/7d3d8bd43f914c7f63f7564e0fb692de31fb5f8e/polaris-react/src/styles/foundation/_layout.scss#L3-L5)

---

That makes sense. Feel free to stay consistent. I would love to clean up all of these values..

---

I think `legacy` is more clear. Thanks for the suggestion 🙌